extensions/interactions/LogicProof/static/js/sharedSpec.js

```diff
@@ -17,7 +17,8 @@
  * interaction.
  */
 
-var errorWrapper = function(dubiousFunction, input, parameter = null) {
+var errorWrapperForSharedSpec = function(
```

Comment:
I think we need to find a way to not have this be in global scope.

Reply:
https://stackoverflow.com/questions/19263586/global-variables-in-karma-test-runner - Can we use the second way to avoid polluting the global scope?

Reply:
Hm, that's a bit indirect.

Here's another idea: I believe you can have nested describe blocks. How about having two layers of describe blocks, and declare this in the outer one?

Reply:
Sounds good, will do.

Reply:
Done.